server/.lsp/config.edn

```diff
@@ -0,0 +1 @@
+{:cljfmt {:indents {#re "io/warn-io" [[:block 1]]}}}
```

Comment:
cc @dwwoelfel looks like clojure-lsp uses clj-fmt to format things. Adding this rule fixed the issue in permissioned-transaction.clj!